### ISSUE TITLE
Rebuild the indexes after changing the current profile

### ISSUE THREAD

**Issue by pfrazee** [beakerbrowser/fritter#4]
Tara's currently adding a tool to change the active profile. We need WebDB to add a 'reset indexes' function, and then we need to call that method after the change.

**Comment by pfrazee** [comment]
Here's how we can do this:

```js
await fritter.db.close()
await WebDB.delete('fritter')
```

That'll clear out the IndexedDB completely. Then you'll want to reload the page.

**Comment by cinnamon-bun** [comment]
Likely related:
When switching profiles, the Notifications tab will continue to show notifications for the profile you just left.

**Comment by pfrazee** [comment]
Yeah there's some kind of funky bug with notifications not always being computed for the correct profile
